habitat/core/embodied_task.py

```diff
@@ -140,6 +230,117 @@ def __init__(
         self._sim = sim
         self._dataset = dataset
 
+        task_measurements = []
+        for measurement_name in config.MEASUREMENTS:
+            measurement_cfg = getattr(config, measurement_name)
+            measure_type = registry.get_measure(measurement_cfg.TYPE)
+            assert (
+                measure_type is not None
+            ), "invalid measurement type {}".format(measurement_cfg.TYPE)
+            print("measurement type {}".format(measurement_cfg.TYPE))
+            task_measurements.append(
+                measure_type(
+                    sim=sim, config=measurement_cfg, dataset=dataset, task=self
+                )
+            )
+        self.measurements = Measurements(task_measurements)
+
+        task_sensors = []
+        for sensor_name in config.SENSORS:
+            sensor_cfg = getattr(config, sensor_name)
+            sensor_type = registry.get_sensor(sensor_cfg.TYPE)
+            assert sensor_type is not None, "invalid sensor type {}".format(
+                sensor_cfg.TYPE
+            )
+            print("sensor type {}".format(sensor_cfg.TYPE))
+            task_sensors.append(
+                sensor_type(sim=sim, config=sensor_cfg, dataset=dataset)
+            )
+
+        self.sensor_suite = SensorSuite(task_sensors)
+
+        # TODO: move sensors and measures to `_init_entities`
```

Comment:
Does this TODO still need to happen?

Reply:
Already finished in next update.

Reply:
I don't see this?  Or has that update not been pushed yet?